spec/content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/killvirtv2v_spec.rb

```diff
@@ -0,0 +1,126 @@
+require_domain_file
+require File.join(ManageIQ::Content::Engine.root, 'content/automate/ManageIQ/Transformation/Common.class/__methods__/utils.rb')
+require File.join(ManageIQ::Content::Engine.root, 'content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/utils.rb')
+
+describe ManageIQ::Automate::Transformation::TransformationHosts::Common::KillVirtV2V do
+  let(:user) { FactoryGirl.create(:user_with_email_and_group) }
+  let(:task) { FactoryGirl.create(:service_template_transformation_plan_task) }
+  let(:host) { FactoryGirl.create(:host) }
+
+  let(:svc_model_user) { MiqAeMethodService::MiqAeServiceUser.find(user.id) }
+  let(:svc_model_task) { MiqAeMethodService::MiqAeServiceServiceTemplateTransformationPlanTask.find(task.id) }
+  let(:svc_model_host) { MiqAeMethodService::MiqAeServiceHost.find(host.id) }
+
+  let(:root) do
+    Spec::Support::MiqAeMockObject.new(
+      'current'             => current_object,
+      'user'                => svc_model_user,
+      'state_machine_phase' => 'cleanup'
+    )
+  end
+
+  let(:current_object) { Spec::Support::MiqAeMockObject.new }
+  let(:ae_service) do
+    Spec::Support::MiqAeMockService.new(root).tap do |service|
+      current_object = Spec::Support::MiqAeMockObject.new
+      current_object.parent = root
+      service.object = current_object
+    end
+  end
+
+  let(:svc_vmdb_handle_host) { MiqAeMethodService::MiqAeServiceHost }
+
+  before do
+    allow(ManageIQ::Automate::Transformation::Common::Utils).to receive(:task).with(ae_service).and_return(svc_model_task)
+    allow(svc_model_task).to receive(:get_option).with(:transformation_host_id).and_return(svc_model_host.id)
+
+    allow(ae_service).to receive(:vmdb).with(:host).and_return(svc_vmdb_handle_host)
+    allow(svc_vmdb_handle_host).to receive(:find_by).with(:id => svc_model_host.id).and_return(svc_model_host)
+  end
+
+  before(:each) do
+    ManageIQ::Automate::Transformation::TransformationHosts::Common::KillVirtV2V.instance_variable_set(:@task, nil)
```

Comment:
`before(:each)` is the default action so it is the same as the `before` block above.  This line can just move up in to the previous block.